deps/Makefile

```diff
@@ -161,6 +161,15 @@ libconfig/libconfig/lib/.libs/libconfig++.a:
 
 libconfig: libconfig/libconfig/lib/.libs/libconfig++.a
 
+prometheus-cpp/lib/libprometheus-cpp-core.a:
+	cd prometheus-cpp && tar --strip-components=1 -zxf v0.9.0.tar.gz -C .
+	cd prometheus-cpp && tar --strip-components=1 -zxf civetweb-v1.11.tar.gz -C 3rdparty/civetweb
+	cd prometheus-cpp && tar --strip-components=1 -zxf googletest-v1.10.0.tar.gz -C 3rdparty/googletest
+	cd prometheus-cpp && patch -p1 < patches/serial_exposer.patch
+	cd prometheus-cpp && cmake . -DBUILD_SHARED_LIBS=OFF
+	cd prometheus-cpp && make -j 4
```

Comment:
do not specify `-j 4`.
This is inherited by the outer `make`.

Also , you shouldn't call `make` directly. Use ` CC=${CC} CXX=${CXX} ${MAKE}` instead